### PR TITLE
New package: libnova-0.15.0

### PR DESCRIPTION
**Pull request by travankor**

### CODE REVIEW

**Comment by maxice8** [comment]
usr/bin/libnovaconfig needs to be in -devel package.

**Comment by maxice8** [comment]
There is a file conflict

**Comment by travankor** [comment]
think it's fixed.

File: srcpkgs/libnova/template [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Template file for 'libnova'
+pkgname=libnova
+version=0.15.0
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf automake libtool"
+short_desc="A double precision astrometry and astrodynamics library"
+maintainer="travankor <travankor@tuta.io>"
+license="LGPL-2.0-only"
```
Comment (maxice8) [inline srcpkgs/libnova/template]: LGPL-2.0-or-later

$ head -n 5 masterdir/builddir/libnova-0.15.0/src/airmass.c
/*
 *  This library is free software; you can redistribute it and/or
 *  modify it under the terms of the GNU Lesser General Public
 *  License as published by the Free Software Foundation; either
 *  version 2 of the License, or (at your option) any later version.

**Pull request merged**